### ISSUE TITLE
feature wishlists 

### ISSUE THREAD

**Issue by jonschlinkert** [jonschlinkert/remarkable#2]
from https://github.com/jonschlinkert/remarked/issues/12#issuecomment-54081910
- [x] <ins>
- [x] <mark>
- [x] <sub>
- [x] <sup>
- [ ] Citations
- [x] Definition lists https://github.com/chjj/marked/issues/254
- [x] Footnotes
- [x] Abbreviations https://michelf.ca/projects/php-markdown/extra/#abbr
- [ ] To-do lists: https://github.com/chjj/marked/issues/107
- [ ] Macros (see https://github.com/jonschlinkert/remarked/issues/9)
  - blocks (with open/close marks or open + indent content)
  - inline (no convention about format).
- [x] Source maps? https://github.com/jgm/stmd/issues/57
- [x] Typographical features: https://github.com/chjj/marked/issues/27. Macros should be able to handle these

---

**upd** by @puzrin:

No questions! No discussions! Only feature name + link to details. This issue is used by devs to keep archtecture in sync with possible needs. If you wish to discuss any feature - just create new a ticket.

**Comment by anaran** [comment]
github task list support is not implemented yet, right?

It's kind of hard to follow the situation about them.

**Comment by felixSchl** [comment]
Yeah what's the story about githab task lists, is this still on the radar?
